modules/spring-config/src/main/java/org/mule/runtime/config/internal/SpringRegistry.java

```diff
@@ -184,16 +184,18 @@ public Object lookupObject(String key, boolean applyLifecycle) {
         }
         return null;
       }
+      applyLifecycleIfPrototype(object, key, applyLifecycle);
```

Comment:
Can you validate that a prototype object created with the SpringConfig does not gets applied lifecycle?. We don't support Initializable, Startable, Stoppable or Disposable in spring module.